pytorch_toolkit/ote/ote/readme.md

```diff
@@ -0,0 +1,201 @@
+# Model Templates
+
+Model Templates defines training procedure and its interface for a given neural network topology.
+
+## Directory structure (openvino_training_extensions/pytorch_toolkit)
+
+Each model template is related to 4 scripts: `train.py`, `eval.py`, `export.py` and `quantize.py`.
+
+```bash
+├── object_detection
+│   ├── face-detection
+│   │   ├── face-detection-0200
```

Comment:
Currently the repo structure is different: `model_templates` after `object_detection`.